static/src/js/util/request/request.js

```diff
@@ -108,6 +108,9 @@ export default class Request {
       method: 'get',
       baseURL: this.baseUrl,
       url,
+      headers: {
```

Comment:
This needs to be surrounded in if statements to restrict it to authenticated requests, currently this will add this for ALL requests.

Reply:
👍